phase1/gce/do

```diff
@@ -8,7 +8,7 @@ set -x
 cd "${BASH_SOURCE%/*}"
 
 generate_token() {
-  perl -e 'printf "%06x:%08x%08x\n", rand(0xffffff), rand(0xffffffff), rand(0xffffffff);'
+  perl -e 'printf "%06x.%08x%08x\n", rand(0xffffff), rand(0xffffffff), rand(0xffffffff);'
```

Comment:
FYI: you _may_ use `kubeadm token generate for this as well, but nothing says you have to